mongo/options/aggregateoptions.go

```diff
@@ -163,3 +164,15 @@ func (ao *AggregateOptionsBuilder) SetCustom(c bson.M) *AggregateOptionsBuilder
 
 	return ao
 }
+
+// SetRawData sets the value for the RawData field. If true, it allows the CRUD operations to access timeseries
+// collections on the bucket-level. This option is only valid for MongoDB versions >= 9.0. The default value is false.
```

Comment:
We should add a note that enabling it on pre-9.0 server versions has no effect. This applies for all `SetRawData` docs.

Note: This comment is irrelevant if we convert RawData into an internal-only option.